bodhi/server/models/models.py

```diff
@@ -1257,6 +1263,7 @@ def send_update_notice(self):
             log.error("Cannot find mailing list address for update notice")
             log.error("release_name = %r", release_name)
 
+
```

Comment:
PEP-8 dictates that there should be 1 space here instead of 2 since this separation is between two methods and not between two top-level objects.